tests/e2e/specs/out_of_game/stats.spec.js

```diff
@@ -0,0 +1,199 @@
+    cy.get('[data-rank=Player1]').contains(1);
+    cy.get("[week-1='Player1']").contains('W: 3, P: 5');
+    cy.get("[week-total='Player1']").contains('W: 6, P: 10');
+    cy.get("[week-1='Player5']").contains('W: 1, P: 3');
+    cy.get('tr.active-user-stats').contains(playerOne.username);
+  });
+
+  it('Filters table to display wins, points, or both', () => {
+    // 16 columns: username, rank, total, + 13 weeks
+    cy.get('th').should('have.length', 16);
+    // Switch to points only
+    cy.get('[data-cy=metric-select]').click({ force: true });
+    cy.contains('Points Only').click();
+    // Only points are displayed
+    cy.get("[week-1='Player1']")
+      .contains('5')
+      .should('not.contain', 'Wins');
+    cy.get('th').should('have.length', 16);
+    // Switch to wins only
+    cy.get('[data-cy=metric-select]').click({ force: true });
+    cy.contains('Wins Only').click();
+    cy.get('th').should('have.length', 16);
+    // Only wins are displayed
+    cy.get("[week-1='Player1']")
+      .contains('3')
+      .should('not.contain', 'Points');
+    cy.get("[points-1='Player1']").should('not.exist');
+  });
+
+  it('Filters table to show selected weeks', () => {
+    // 16 columns: username, rank, total, 13 weeks
+    cy.get('th').should('have.length', 16);
+    // Total counts across all weeks
+    cy.get('[week-total=Player1]').should('contain', 'W: 6, P: 10');
+    // Deselect every week except week 1
+    cy.get('[data-cy=week-select]').click({ force: true });
+    cy.get('[role=option]')
+      .contains('Week 2')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 3')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 4')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 5')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 6')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 7')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 8')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 9')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 10')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 11')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 12')
+      .click();
+    cy.get('[role=option]')
+      .contains('Week 13')
+      .click();
+    cy.get('body').type('{esc}');
+
+    // Expect 5 columns: username, rank, total, week_1
+    cy.get('th').should('have.length', 4);
+    // Total counts should only consider the selected weeks
+    cy.get('[week-1=Player1]').should('contain', 'W: 3, P: 5');
+    cy.get('[week-total=Player1]').should('contain', 'W: 6, P: 10');
+  });
+
+  it('Selects different seasons to show their results', () => {
+    // Three award cards for Clubs 2022
+    cy.get('[data-tournament]').should('have.length', 3);
+    // Switch season to diamonds 2022
+    cy.get('[data-cy=season-select]').click({ force: true });
```

Comment:
Need to force because vuetify injects attributes onto the nested input inside the select which isn't actually clickable because it's covered by other injected content. This is awkward but works well enough I think